services/src/test/scala/cromwell/pipeline/service/ProjectFileServiceTest.scala

```diff
@@ -0,0 +1,39 @@
+package cromwell.pipeline.service
+
+import java.nio.file.Paths
+
+import cromwell.pipeline.datastorage.dao.repository.utils.TestProjectUtils
+import cromwell.pipeline.datastorage.dto.ProjectFile
+import org.mockito.Mockito.when
+import org.scalatest.{ AsyncWordSpec, Matchers }
+import org.scalatestplus.mockito.MockitoSugar
+
+import scala.concurrent.Future
+
+class ProjectFileServiceTest extends AsyncWordSpec with Matchers with MockitoSugar {
+
+  private val projectVersioning = mock[ProjectVersioning[VersioningException]]
+  private val projectFileService = new ProjectFileService(projectVersioning)
+
+  "ProjectFileServiceTest" when {
+    // not sure about this tests
```

Comment:
You test not only update file here but projectFileService(35 line), so, that's all right, since you have the only one method in ProjectFileService.